src/bin/cratesfyi.rs

```diff
@@ -1,17 +1,20 @@
 use std::env;
 use std::path::PathBuf;
+use std::sync::Arc;
 
 use cratesfyi::db::{self, add_path_into_database, connect_db};
 use cratesfyi::utils::{add_crate_to_queue, remove_crate_priority, set_crate_priority};
 use cratesfyi::Server;
 use cratesfyi::{DocBuilder, DocBuilderOptions, RustwideBuilder};
+use failure::Error;
 use structopt::StructOpt;
 
-pub fn main() {
+pub fn main() -> Result<(), Error> {
     let _ = dotenv::dotenv();
     logger_init();
 
-    CommandLine::from_args().handle_args();
+    CommandLine::from_args().handle_args()?;
+    Ok(())
```

Comment:
nit: I think this looks nicer.
```suggestion
    CommandLine::from_args().handle_args()
```